### ISSUE TITLE
Bodhi updates for development branches show as "skipped" and are not linked.

### ISSUE THREAD

**Issue by ckelleyRH** [packit/dashboard#379]
### What happened? What is the problem?

Bodhi updates for development branches (`f40` and `rawhide` at the time of writing) are displayed as "skipped" in the new Bodhi Update job section added in #215: https://dashboard.packit.dev/jobs/bodhi-updates

I'm not sure what "skipped" means in context for Packit but, for example:
https://dashboard.packit.dev/results/bodhi-updates/266
https://dashboard.packit.dev/results/bodhi-updates/265

Alias "not provided" and Update Creation Time "not provided", but the Bodhi updates are actually created by Packit:
https://bodhi.fedoraproject.org/updates/FEDORA-2024-289a641391
https://bodhi.fedoraproject.org/updates/FEDORA-2024-75d641297c

### What did you expect to happen?

Development branch updates come through with "success" and a link to the Bodhi update is provided.

### Example URL(s)

https://dashboard.packit.dev/jobs/bodhi-updates

### Steps to reproduce

```bash
1. GOTO https://dashboard.packit.dev/jobs/bodhi-updates
2. Inspect any `f40` or `rawhide` update
3.
```

### Workaround

- [ ] There is an existing workaround that can be used until this issue is fixed.

### Participation

- [X] I am willing to submit a pull request for this issue. (Packit team is happy to help!)

**Comment by ckelleyRH** [comment]
Well, I said there is no workaround but one can simply go to Bodhi and look for the update (which is what I did). I knew it existed because the version bump Bugzilla was updated with links to both updates: https://bugzilla.redhat.com/show_bug.cgi?id=2265008

**Comment by lbarcziova** [comment]
hi @ckelleyRH ! Thanks for this issue.

> but the Bodhi updates are actually created by Packit

Even though Bodhi UI shows that they were created by packit user, those are actually triggered automatically, see [the docs](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html#associate-bugs-to-automatic-updates). Therefore from Packit side, there is really no submission time or anything. You can also see `This update was automatically created` in the update detail you linked.

I get your point it might not be clear looking at our dashboard and am wondering what could we improve since ideally, we want to display only actions that were really triggered/processed by Packit. We could definitely at least provide this explanation in the dashboard.

**Comment by ckelleyRH** [comment]
> I get your point it might not be clear looking at our dashboard and am wondering what could we improve since ideally, we want to display only actions that were really triggered/processed by Packit. We could definitely at least provide this explanation in the dashboard.

If we're saying that we only want to show updates that were _actually_ generated by Packit, rather than just looking like they were to the untrained observer, then perhaps development branch updates shouldn't be displayed at all? I wouldn't have thought about this at all probably if they weren't displayed, I would have just seen the bug closed due to the update going straight to stable and moved on.

**Comment by ckelleyRH** [comment]
> We usually suggest to put in the `bodhi_update` job definition the `fedora-branched` alias as a target. Because bodhi updates are triggered automatically for the development branches. And for this reason we got an error.

Yes, I can go through and update all the configurations so that this doesn't happen at all, thanks for the pointer!

**Comment by ckelleyRH** [comment]
@majamassarini - does `fedora-branched` include `f40` (I expect it does)? If so, even if I fix my configs to not build for `rawhide`, it will still trigger this behaviour for `f40` right?

**Comment by majamassarini** [comment]
yes, it should include f40. This behaviour puzzled me too (the automatic update for f40). I think this could be something temporary since the branching has happened not so long ago. But let's see what others say (@lbarcziova or @lachmanfrantisek).

**Comment by lbarcziova** [comment]
Yes, this should be only temporary, until it reaches [Updates-testing Activation](https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/#updates-testing-activation), after that, the updates for f40 will not be created automatically.

**Comment by ckelleyRH** [comment]
> Yes, this should be only temporary, until it reaches [Updates-testing Activation](https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/#updates-testing-activation), after that, the updates for f40 will not be created automatically.

Currently I have a basically identical Packit set up in every repo like this:
```
jobs:
- job: pull_from_upstream
  trigger: release
  dist_git_branches:
    - fedora-development
 
- job: koji_build
  trigger: commit
  dist_git_branches:
    - fedora-development
 
- job: bodhi_update
  trigger: commit
  dist_git_branches:
    - fedora-development
```
So I want to only do rebases in `rawhide`and if there is an embryonic stable branch like `f40` is now, I want that rebase in that branch too - hence `fedora-development`.

If I am to not create the Bodhi update for `rawhide`, which I agree is unnecessary, to what can I change `fedora-development`? There doesn't appear to be a sensible way of doing this...

**Comment by majamassarini** [comment]
> If I am to not create the Bodhi update for `rawhide`, which I agree is unnecessary, to what can I change `fedora-development`? There doesn't appear to be a sensible way of doing this...

I am not sure I am following you. I think you simply don't need a Packit `bodhi_update` job.

**Comment by ckelleyRH** [comment]
> > If I am to not create the Bodhi update for `rawhide`, which I agree is unnecessary, to what can I change `fedora-development`? There doesn't appear to be a sensible way of doing this...
> 
> I am not sure I am following you. I think you simply don't need a Packit `bodhi_update` job.

I think I may be just confused, but it doesn't matter anyway because the team has decided to use `fedora-rawhide` and drop `bodhi-update` so this all goes away for us. Thanks for answering the scope creep questions (sorry)!
